src/tiledimage.js

```diff
@@ -325,7 +344,8 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
      * @returns {Boolean} whether the item still needs to be drawn due to blending
      */
     setDrawn: function(){
-        this._needsDraw = this._isBlending || this._wasBlending;
+        this._needsDraw = this._isBlending || this._wasBlending ||
+            (this.opacity > 0 && this._lastDrawn.length < 1);
```

Comment:
What cases are you guarding against by adding this extra condition? Are there times where this might cause `_needsDraw` to stay `true` unnecessarily?

Reply:
I don't actually remember. I can try removing this and see how it behaves. I agree even if this check is true it should not keep it on true for too long... Maybe it is an artifact from the old logics. Good catch!